tests/PHPStan/Analyser/data/unresolvable-types.php

```diff
@@ -5,12 +5,10 @@
 /**
  * @param array<int, int, int> $arrayWithTooManyArgs
  * @param iterable<int, int, int> $iterableWithTooManyArgs
- * @param \Foo<int> $genericFoo
```

Comment:
I think this should still fail in this PR, there's no GenericObjectType implementation yet...